{{cookiecutter.app_name}}/config/app.toml

```
@@ -0,0 +1,5 @@
+[debug]
+database_url = "postgres://postgres:postgres@0.0.0.0:5432/{{cookiecutter.app_name}}_dev?sslmode=disable"
```

Is there a missing space and double quote?

```suggestion
database_url = "postgres://postgres:postgres@0.0.0.0:5432/{{cookiecutter.app_name}}_dev?sslmode = "disable"
```

---

No, that is the single string
`"postgres://postgres:postgres@0.0.0.0:5432/{{cookiecutter.app_name}}_dev?sslmode=disable"`

But I change the way to get `database_url` in 190a010690dd634483660246d2f160b13ef11f67

---

Oh indeed. The line wrap was awkward there 🙈